app/frontend/src/features/communities/events/EventCard.tsx

```diff
@@ -48,9 +49,23 @@ const useStyles = makeStyles<Theme, { eventImageSrc: string }>((theme) => ({
   },
   title: {
     ...theme.typography.h2,
-    height: `calc(2 * calc(${theme.typography.h2.lineHeight} * ${theme.typography.h2.fontSize}))`,
+    height: `calc(3 * calc(${theme.typography.h2.lineHeight} * ${theme.typography.h2.fontSize}))`,
```

Comment:
This should still be 2, no?

Reply:
> For some weird reasons, on small mobile screens sometimes the event title goes onto 3 lines. I think the LineEllipsis thing is kinda flaky

That's why it's 3. See the before screenshot for mobile when it's 2

Reply:
It's not flaky, it just only updates on load, not resize. We could try to change that or just decide to not worry about viewport size changes, but I think it should be 2 here

Reply:
Even with 3 * line height, I get this on refresh (initial load, no orientation changes) using the dev tools on iPhone 6/7/8 Plus screen size from the dev tool:
![Screenshot 2021-08-29 at 23 11 46](https://user-images.githubusercontent.com/13669362/131266973-9adb6552-624c-433e-b401-83c484a15de0.png)

Maybe we need to truncate by character counts as well. Or re-think the design slightly, putting a fixed height on dynamic content is a bad pattern imo.